packages/web3-eth-personal/test/integration/personal.test.ts

```diff
@@ -117,45 +111,43 @@ describe('personal integration tests', () => {
 		const account = await ethPersonal.newAccount('cde');
 		const updatedAccountList = await ethPersonal.getAccounts();
 		accountList.push(account);
-		expect(updatedAccountList).toStrictEqual(accountList);
+		expect(updatedAccountList.length).toBeGreaterThan(account.length);
```

Comment:
Is this a good suggestion?:
```suggestion
		const accountsList = await ethPersonal.getAccounts();
		accountsList.push(account);
		const updatedAccountsList = await ethPersonal.getAccounts();
		expect(updatedAccountsList.length).toStrictEqual(accountsList.length + 1);
```

Reply:
Also, when I run parallel tests, a few tests add accounts. And it will always be more than  +1.